backend/src/models/ArtistManager.js

```diff
@@ -5,9 +5,11 @@ class ArtistManager extends AbstractManager {
 
   insert(artist) {
     return this.connection.query(
-      `insert into ${ArtistManager.table} (user_id,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,role,cachet) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
+      `insert into ${ArtistManager.table} (user_id,status,siren,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,cachet) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
       [
         artist.user_id,
+        artist.status,
+        artist.siren,
```

Comment:
- 2 field names :heavy_check_mark: 
- 2 variables :heavy_check_mark: 
- 1 question mark ? Has it been tested ? :thinking:

Reply:
Yup it's work